src/engraving/rendering/dev/slurtielayout.cpp

```diff
@@ -2446,11 +2446,15 @@ void SlurTieLayout::layoutSegment(SlurSegment* item, LayoutContext& ctx, const P
 
 void SlurTieLayout::computeMidThickness(SlurTieSegment* slurTieSeg, double slurTieLengthInSp)
 {
+    const Millimetre endWidth = slurTieSeg->isTie() ? slurTieSeg->style().styleMM(Sid::TieEndWidth)
```

Comment:
Perhaps this needs to be `isTieSegment` instead of `isTie`... because `isTie()` means `elementType() == ElementType::TIE`, while it is `ElementType::TIE_SEGMENT` for tie segments.

Reply:
That seems to have done the trick, thanks!